src/ResourceManager/Network/Commands.Network/VirtualNetwork/Subnet/AzureVirtualNetworkSubnetConfigBase.cs

```diff
@@ -58,5 +59,11 @@ public class AzureVirtualNetworkSubnetConfigBase : NetworkBaseCmdlet
             ParameterSetName = "SetByResource",
             HelpMessage = "RouteTable")]
         public PSRouteTable RouteTable { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Private Access Service")]
+        public List<string> PrivateAccessService { get; set; }
```

Comment:
It was decided to skip adding of any validation sets on Swagger/clients level as guys working on this feature would like to keep Azure PS working w/o updates after they add new supported values